test/cli/dev.js

```diff
@@ -32,6 +32,25 @@ test('`fusion dev` works', async t => {
   t.end();
 });
 
+test('`fusion dev --dir` works w/ relative dir', async t => {
```

Comment:
I think relative dir should be explicitly disallowed. Or alternatively, it should be immediately converted to an absolute path as soon as it is provided

Reply:
In our own code, we should be able to assume dir is an absolute path and as a replacement for cwd.

Reply:
From what I could tell, we do convert it to a absolute path when we spawn the child process in the dev runtime, but it's common for people to pass in a relative path to the CLI arg